package.json

```diff
@@ -106,7 +106,6 @@
     "@types/uuid": "^8.3.0",
     "@typescript-eslint/eslint-plugin": "^5.37.0",
     "@typescript-eslint/parser": "^5.37.0",
-    "cypress": "^10.7.0",
```

Comment:
why did you remove cypress?

Reply:
I installed cypress when I was working on the code coverage issue, then I had some issues running the Chris app cause I wasn't able to set it up, so I had to uninstall it so I could work on other issues. Initially, cypress wasn't installed, I had to install it so I could run the cypress script command in package.json